tests/conftest.py

```diff
@@ -13,10 +13,16 @@
 @pytest.fixture(autouse=True)
 # Global context and logging levels reset is required.
 # Without it, state from previous tests is visible in following tests.
-def reset_global_context_and_logging_levels_after_each_test(request):
+#
+# This automatically used setup fixture is required to use test.conf from resources
+# in unit tests which are not using "runner" fixture (tests which do not invoke CLI command).
+def reset_global_context_and_setup_config_and_logging_levels(
+    request, test_snowcli_config
+):
     cli_context_manager.reset()
     cli_context_manager.set_verbose(False)
     cli_context_manager.set_enable_tracebacks(False)
+    config_init(test_snowcli_config)
```

Comment:
I needed to move this line above `loggers.create_loggers` - otherwise it would use config from the previous test, throwing "logs directory not found" error